docs/installation/installation.md

```diff
@@ -1,12 +1,152 @@
+used as Django settings.
+
+### Preparation of Vagrant box
+
+Create and enter a new parent directory for your Vagrant box:
+
+    mkdir ralph_vagrant
+    cd ralph_vagrant
+
+Create Vagrant file in the parent directory:
+
+    cat -> Vagrantfile
+    Vagrant.configure("2") do |config|
+      config.vm.box = "ubuntu/bionic64"
+      config.vm.hostname = "ralph"
+      config.vm.network "forwarded_port", guest: 80, host: 8000
+      config.vm.provider "virtualbox" do |vb|
+        vb.memory = "2048"
+      end
+    end
+    ^D
+
+Start and enter the box:
+
+    vagrant up
+    vagrant ssh
+
+### Ralph installation
+
+    sudo apt-key adv --keyserver  hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
+    sudo sh -c "echo 'deb https://dl.bintray.com/vi4m/ralph bionic main' >  /etc/apt/sources.list.d/vi4m_ralph.list"
+    sudo apt-get update
+    sudo apt-get install mysql-server nginx ralph-core
+
+When prompted, input Ralph database settings. For testing purposes, chosing the
+default settings will be fine. You can review the settings later in
+`/etc/ralph/conf.d/database.conf`.
+
+### Nginx configuration
+
+Configure nginx by editing /etc/nginx/sites-available/default file and pasting
+the following:
+
+    server {
+
+        listen 80;
+        client_max_body_size 512M;
+
+        proxy_set_header Connection "";
+        proxy_http_version 1.1;
+        proxy_connect_timeout  300;
+        proxy_read_timeout 300;
+
+        access_log /var/log/nginx/ralph-access.log;
+        error_log /var/log/nginx/ralph-error.log;
+
+        location /static {
+            alias /usr/share/ralph/static;
+            access_log        off;
+            log_not_found     off;
+            expires 1M;
+        }
+
+        #location /media {
+        #    alias /var/local/ralph/media;
+        #    add_header Content-disposition "attachment";
+        #}
+
+        location / {
+            proxy_pass http://127.0.0.1:8000;
+            include /etc/nginx/uwsgi_params;
+            proxy_set_header Host $http_host;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        }
+    }
+
+After that, restart nginx:
+
+    sudo systemctl restart nginx.service
+
+### Database configuration
+
+Once Ralph is installed, you can configure the database and create a Ralph
+superuser:
+
+    echo "create database ralph_ng" | sudo mysql
+    echo "create user 'ralph_ng'@'localhost' identified by 'ralph_ng'" | sudo mysql
+    echo "grant all privileges on ralph_ng.* to 'ralph_ng'@'localhost'" | sudo mysql
+    sudo ralphctl migrate
+    sudo ralphctl createsuperuser
+
+Pouplate the database with some demo data:
+
+    sudo ralphctl demodata
+
+### Starting Ralph
+
+Now just a finishing touch:
+
+    sudo ralphctl sitetree_resync_apps
+    sudo systemctl enable ralph.service
+    sudo systemctl start ralph.service
+
+And you are all set. Navigate to your new Ralph installation:
+[http://localhost:8000](http://localhost:8000)
+
```

Comment:
### Troubleshooting 

Where to look for Ralph logs when not starting correctly? 
...